src/styles/variables.scss

```diff
@@ -0,0 +1,9 @@
+
+$month-length: 31;
+$week-day: (mon:0, tue:1, wed:2, thu:3, fri:4, sat:5, sun:6);
+
+$block-part: 100px;
+$block-gap : 1px;
+$calendar-padding: 10px;
+
+$container-width: $block-part * 7 + $block-gap * 6 + $calendar-padding;
```

Comment:
You have paddings on two sides of calendar, so you shoul writh $calendar-padding * 2

Reply:
Okey then, but it works good even without any padding calc